src/ui/ui_main.c

```diff
@@ -250,6 +251,26 @@ void _UI_DrawRect(float x, float y, float width, float height, float size, const
 	trap_R_SetColor(NULL);
 }
 
+/**
+ * @brief UI_DrawBox
+ * Draw a filled rectangle.
+ * Coordinates are 640*480 virtual values
+ * @param[in] x
+ * @param[in] y
+ * @param[in] width
+ * @param[in] height
+ * @param[in] color
+ */
+void _UI_DrawBox(float x, float y, float width, float height, const float *color)
+{
+	trap_R_SetColor(color);
+
+	UI_AdjustFrom640(&x, &y, &width, &height);
+	trap_R_DrawStretchPic(x, y, width, height, 0, 0, 0, 0, uiInfo.uiDC.whiteShader);
+
+	trap_R_SetColor(NULL);
+}
+
```

Comment:
`UI_FillRect` (`DC->fillRect`) is line by line identical to this.

https://github.com/etlegacy/etlegacy/blob/ac4b2deeccd6cdbb9cea4e6ba87400dbf602ab87/src/ui/ui_atoms.c#L309-L317

Reply:
Total oversight, thanks! Fixed